packages/shared/src/server/repositories/scores.ts

```diff
@@ -0,0 +1,104 @@
+import { ScoreDataType, ScoreSource } from "@prisma/client";
+import { queryClickhouse } from "./clickhouse";
+
+export type FetchScoresReturnType = {
+  id: string;
+  timestamp: string;
+  project_id: string;
+  trace_id: string;
+  observation_id: string | null;
+  name: string;
+  value: number;
+  source: string;
+  comment: string | null;
+  author_user_id: string | null;
+  config_id: string | null;
+  data_type: string;
+  string_value: string | null;
+  queue_id: string | null;
+  created_at: string;
+  updated_at: string;
+  event_ts: string;
+  is_deleted: number;
+  projectId: string;
+};
+
+const convertToScore = (row: FetchScoresReturnType) => {
```

Comment:
How about we use the results as is and skip the conversion?

Reply:
Unfortunately not possible as we somehow need to map casing and names